pex/bin/pex.py

```diff
@@ -457,9 +457,10 @@ def interpreter_from_options(options):
     return interpreter
 
 
-def build_pex(args, options, resolver_option_builder):
-  with TRACER.timed('Resolving interpreter', V=2):
-    interpreter = interpreter_from_options(options)
+def build_pex(args, options, resolver_option_builder, interpreter=None):
```

Comment:
ah, assuming you meant `bazel`, I seem to get a failure just trying to get to an initial baseline run:

```
[illuminati heron (master)]$ git pull
Already up-to-date.
[illuminati heron (master)]$ git rev-parse HEAD
4a61bae0ac030f75c24e7377ef01ffbe9e495ca2
[illuminati heron (master)]$ bazel build --config=darwin heron/...
INFO: Found 335 targets...
ERROR: /Users/kwilson/dev/heron/heron/shell/src/python/BUILD:17:1: null failed: _pex failed: error executing command bazel-out/local_darwin-fastbuild/bin/third_party/pex/_pex --entry-point heron.shell.src.python.main bazel-out/local_darwin-fastbuild/bin/heron/shell/src/python/heron-shell.pex ... (remaining 1 argument(s) skipped): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
pex options: {'use_wheel': False, 'inherit_path': False, 'script': None, 'python': '/usr/bin/python2.7', 'repos': [PyPIFetcher('https://pypi.python.org/simple/')], 'pex_name': None, 'pypi': True, 'requirement_files': [], 'platform': 'macosx-10.11-intel', 'cache_dir': '/Users/kwilson/.pex/build', 'entry_point': 'heron.shell.src.python.main', 'verbosity': 0, 'zip_safe': True, 'interpreter_cache_dir': '/Users/kwilson/.pex/interpreters', 'ignore_errors': False, 'cache_ttl': None, 'python_shebang': None, 'always_write_cache': False, 'find_links': ''}
pex requirements: ['requests==2.3.0', 'tornado==4.0.2']
Traceback (most recent call last):
  File "/private/var/tmp/_bazel_kwilson/248c4d7c0e2d8586f9fa08f1b261a0ae/heron/bazel-out/local_darwin-fastbuild/bin/third_party/pex/_pex.runfiles/third_party/pex/_pex.py", line 208, in <module>
    sys.exit(main())
  File "/private/var/tmp/_bazel_kwilson/248c4d7c0e2d8586f9fa08f1b261a0ae/heron/bazel-out/local_darwin-fastbuild/bin/third_party/pex/_pex.runfiles/third_party/pex/_pex.py", line 172, in main
    os.path.join(pex_builder.BOOTSTRAP_DIR, 'pkg_resources.py'))
  File "/Users/kwilson/dev/heron/third_party/pex/pex/pex_builder.py", line 138, in add_source
    self._copy_or_link(filename, env_filename, "source")
  File "/Users/kwilson/dev/heron/third_party/pex/pex/pex_builder.py", line 340, in _copy_or_link
    self._chroot.link(src, dst, label)
  File "/Users/kwilson/dev/heron/third_party/pex/pex/common.py", line 267, in link
    os.link(abs_src, abs_dst)
OSError: [Errno 2] No such file or directory
INFO: Elapsed time: 5.105s, Critical Path: 4.79s
```